npm/commander/2.19.0/@n4jsd/commander/index.n4jsd

```diff
@@ -0,0 +1,19 @@
+/**
+ * @see https://github.com/tj/commander.js/
+ */
+export external public class __Command {
```

Comment:
I think this should be `class Command` right away.

Reply:
done